### ISSUE TITLE
Drop island_region and state_province tables.

### ISSUE THREAD

**Issue by ima-hima** [ima-hima/PRIMO#16]
They're unused, so they just add needless complication.

Note that `state_province` points to `country`, so we'll have to pull some data by hand and change the FK in locality. May have to postpone `state_province` removal till after #11 .

- [x] Island region
- [x] State province

**Comment by ima-hima** [comment]
Done in #38 .
